src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -151,3 +151,22 @@ class NoTemplatesServiceApi(object):
             self.stub.Increment, settings=settings)
         api_call(req, metadata=self._headers)
 
+============== file: no_templates_service_client_config.json ==============
+{
+  "interfaces": {
+    "google.example.v1.NoTemplatesService": {
+      "retry_codes": {
+        retry_codes_def:
```

Comment:
How so? This test has a sort of degenerate GAPIC config, so I think something like this might be expected...